pages/rdm_cycle/analysing.md

```diff
@@ -1,35 +1,31 @@
+With many disciplines becoming data-oriented, more and more data intensive projects will occur and will require experts from many thematic fields.
 
 ## What should be considered for data analysis?
+Because of the diversity of domains and techologies in Life Sciences, data can be either "small" or "big data". As a consequence, the methods and technical solutions used for data analysis might differ. The characteristics of "big data" are often summarized by a growing list of ["V's" properties: Volume, Velocity, Variety, Variability, Veracity, Visualization and Value](https://bigdatapath.wordpress.com/2019/11/13/understanding-the-7-vs-of-big-data/).
 
-Because of their nature, data in Life Sciences are now considered as Big Data. These characteristics of Big Data, often summarized by a growing list of "V" properties (Volume, Velocity, Variety, Veracity, Value, etc.), impact strongly the methods and technical solutions used for Data Analysis.
-
-* At the storage level, including the transfer of the data from the data production facility to the computing facility for its analysis, it is worthwhile to compare the cost of the transfer of massive amounts of data compared to the transfer of virtual images of machines for the analysis.
-* The variety of the data poses an integration challenge that can only adressed with the help of best practices that make data interoperable and reusable.
-* The Data Analysis phase relies on the previous steps (collection, processing) that will lay the foundations for the generation of new knowledge by providing acurate and trustworthy data.
-* For the analysis of data, you will first have to consider the computing environment and choose between several computing infrastructure types for e.g. cluster, cloud. You will also need to select the appropriate work environment according to your needs and expertise (command line, web portal).
-* The location of your data is important because of the needed need of proximity with computing resources. This can impact data transfer across the different infrastructures.
-* You will have to select the tools best suited for the analysis of your data. Resources such as [bio.tools](https://bio.tools) can be very helpful.
+* The data analysis stage relies on the previous stages (collection, processing) that will lay the foundations for the generation of new knowledge by providing acurate and trustworthy data.
+* The variety of the data poses an integration challenge that can only be adressed with the help of best practices that make data interoperable and reusable during the collection and/or process stage.
```

Comment:
I'm not sure this should be a consideration for analysis stage

Reply:
Maybe, but I don't think it harms to have it there.

Thanks for the suggestion, are you going to make these changes in a PR and merge it?

Reply:
I'm inclined to delete it.